apps/evi/agent/lib/blob.test.ts

```
@@ -62,3 +62,48 @@ describe('MAX_IMAGE_BYTES', () => {
     expect(MAX_IMAGE_BYTES).toBe(8 * 1024 * 1024)
   })
 })
+
+describe('uploadSandboxImage', () => {
+  const png = new Uint8Array([
+    0x89, 0x50, 0x4E, 0x47, 0x0D, 0x0A, 0x1A, 0x0A,
```

ESLint

[error] There should be a linebreak after this element.

(array-element-newline)